app/assets/javascripts/polaris_view_components.js

```diff
@@ -174,16 +174,20 @@ function formatBytes(bytes, decimals) {
 }
 
 class Autocomplete extends Controller {
-  static targets=[ "popover", "input", "results", "option", "emptyState" ];
+  static targets=[ "popover", "input" ];
   static values={
     url: String,
-    selected: Array
+    selected: Array,
+    selectEventRef: String
   };
   connect() {
     this.inputTarget.addEventListener("input", this.onInputChange);
+    this.boundSelect = this.select.bind(this);
```

Comment:
You could avoid having to do this by declaring the function like this:

```js
select = (event) => {
```

(This autobinds the function so `this` remains the actual controller)